hedera-node/src/main/java/com/hedera/services/state/submerkle/ExpirableTxnRecord.java

```diff
@@ -75,7 +75,8 @@
 	static final int RELEASE_0180_VERSION = 5;
 	static final int RELEASE_0210_VERSION = 6;
 	static final int RELEASE_0230_VERSION = 7;
-	static final int MERKLE_VERSION = RELEASE_0230_VERSION;
+	static final int RELEASE_0250_VERSION = 8;
```

Comment:
Nit: Prefer this be called RELEASE_0240_OR_0250_VERSION = 8;   

And then add comment explaining why it's okay to combine the two release version numbers.